src/util.test.js

```diff
@@ -318,3 +320,18 @@ describe('shouldConfirmStatusModalBeShown', () => {
     });
   });
 });
+
+describe('isDcbUser', () => {
+  it('should return true when user has lastName as "DcbSystem"', () => {
+    const user = {
+      lastName: DCB_USER.lastName
+    };
+    expect(isDcbUser(user)).toBeTruthy();
+  });
+  it('should return false when user does not have lastName as "DcbSystem"', () => {
```

Comment:
Could you please add empty line before `it`?

Reply:
done